src/sql/plugins/wiki/plugin/create_db_wiki.sql

```diff
@@ -27,6 +31,8 @@ CREATE TABLE wiki_topic_version (
   date_edition TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
   id_topic_version_previous INT DEFAULT '0' NOT NULL,
   wiki_content LONG VARCHAR,
+  is_published INT DEFAULT '0' NOT NULL,
+  background_color VARCHAR(50),

```

Comment:
a more generic field like "style" or "class" could have been more generic

Reply:
oups, this field wasn't for this branch, so it's deleted now